### PR TITLE
PG-541: Clear form and show progress when opening project

### PR DESCRIPTION
**Pull request by phillip-hopper**
Resolves #PG-541

[Review on Reviewable](https://reviewable.io/reviews/sillsdev/glyssen/73)

### CODE REVIEW

File: Glyssen/MainForm.cs [hunk 1]
```diff
@@ -459,12 +523,7 @@ private void SetupUiLanguageMenu()
 
 		private void Assign_Click(object sender, EventArgs e)
 		{
-			using (var viewModel = new AssignCharacterViewModel(m_project))
```
Comment (andrew-polk) [inline Glyssen/MainForm.cs]: Was this removed on purpose?

File: Glyssen/MainForm.cs [hunk 2]
```diff
@@ -172,24 +180,69 @@ private void MainForm_Load(object sender, EventArgs e)
...
+			m_progressBar.Visible = false;
+			m_tableLayoutPanel.Enabled = true;
+			UpdateDisplayOfProjectInfo();
+		}
+
+		private async void LoadProject(string filePath)
+		{
+			InitializeProgress(3);
+			IncrementProgress();
+			await Task.Delay(300);
```
Comment (andrew-polk) [inline Glyssen/MainForm.cs]: Do these calls actually make it take longer to load a project?  If so, would we be better off just having a marquee progress bar which goes until the load is complete?

**Comment by andrew-polk** [comment]
When I ran it, it looks like the label and all the buttons got moved down a little bit on the Main Form.  Now it looks like there is extra space at the top.

File: Glyssen/MainForm.cs (hunk 1 above)
Comment (phillip-hopper) [inline Glyssen/MainForm.cs]: Fixed.

File: Glyssen/MainForm.cs (hunk 2 above)
Comment (phillip-hopper) [inline Glyssen/MainForm.cs]: Removed the delays and made changed to marque progress bar.

**Comment by phillip-hopper** [comment]
Removed most of the top padding.

**Comment by phillip-hopper** [comment]
Removed the progress bar.

File: Glyssen/MainForm.cs [hunk 3]
```diff
@@ -172,6 +187,20 @@ private void MainForm_Load(object sender, EventArgs e)
 				LoadProject(Settings.Default.CurrentProject);		
 		}
 
+		private void InitializeProgress()
+		{
+			ResetUi();
+			m_tableLayoutPanel.Enabled = false;
+			Cursor = Cursors.WaitCursor;
```
Comment (andrew-polk) [inline Glyssen/MainForm.cs]: It appears this should be 
Cursor.Current = Cursors.WaitCursor;

The way it is now, it seems the UI still never updates with the correct cursor.

File: Glyssen/MainForm.cs [hunk 4]
```diff
@@ -172,6 +187,20 @@ private void MainForm_Load(object sender, EventArgs e)
...
+		}
+
+		private void CloseProgress()
+		{
+			Cursor = Cursors.Default;
```
Comment (andrew-polk) [inline Glyssen/MainForm.cs]: See comment above.

File: Glyssen/MainForm.cs (hunk 3 above)
Comment (phillip-hopper) [inline Glyssen/MainForm.cs]: I made the change, although I didn't notice any difference in behavior.

**Pull request merged**